### PR TITLE
Merge remote-tracking branch 'origin/alaska-prod' into alaska-alt-1

### PR DESCRIPTION
**Pull request by jovial**

### CODE REVIEW

**Comment by jovial** [comment]
Dammit - ended up committing changes I didn't want.

**Comment by jovial** [comment]
Turns out that I had selected the wrong base for the PR. Ready to review.

File: etc/kayobe/opensm.yml [hunk 1]
```diff
@@ -4,7 +4,7 @@
 
 # Whether OpenSM is enabled.
 #opensm_enabled:
-opensm_enabled: False
+opensm_enabled: True
```
Comment (jovial) [inline etc/kayobe/opensm.yml]: Is the subnet manager shared across production and alt-1?

Comment (markgoddard) [reply etc/kayobe/opensm.yml]: Yes - just one IB network

**Comment by markgoddard** [comment]
Why are we merging prod into alt-1? Normally we do the reverse.

File: etc/kayobe/kolla/inventory/overcloud-components.j2 [hunk 2]
```diff
@@ -0,0 +1,280 @@
+# This inventory section provides a mapping of the components to top level
```
Comment (markgoddard) [inline etc/kayobe/kolla/inventory/overcloud-components.j2:1]: Sure we want this? alt-1 monitoring is on the controller, prod monitoring is on a separate host.

**Comment by jovial** [comment]
> Why are we merging prod into alt-1? Normally we do the reverse.

So that we test an upgrade with equivalent config. Some stuff has been committed to production that is not in alt-1.

File: etc/kayobe/kolla/inventory/overcloud-components.j2 (hunk 2 above)
Comment (jovial) [reply etc/kayobe/kolla/inventory/overcloud-components.j2:1]: we define:
```
kolla_overcloud_inventory_top_level_group_map:
  control:
    groups:
      - controllers
  network:
    groups:
      - network
  compute:
    groups:
      - compute
  monitoring:
    groups:
      - controllers
```
https://github.com/SKA-ScienceDataProcessor/alaska-kayobe-config/blob/29429c5264eb0d6eed20b668231bc7ac357d23b3/etc/kayobe/kolla.yml#L214

So it should be equivalent, right? My opinion is that the more similar the config is the better.

**Review by markgoddard — APPROVED** [review]

**Pull request merged**